create_vm.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+# curl https://raw.githubusercontent.com/jamorham/nightscout-vps/vps-dev/create_vm.sh | bash
+
+if [[ "$CLOUD_SHELL" != true ]]; then
+  echo "You can only run this script in Cloud Shell."
+  echo "Terminating."
+  exit 0
+fi
+
+echo
+echo "🌩️  Google Cloud Nightscout VM Installer"
+echo "----------------------------------------"
+echo "✅ Environment: Cloud Shell confirmed"
+echo
+
+# --- Check for existing VMs ---
+existing_vms=$(gcloud compute instances list --format="value(name)")
+
+if [[ -n "$existing_vms" ]]; then
+  echo "⚠️  Existing virtual machines found:"
+  echo "$existing_vms"
+  echo
+  read -p "Do you still want to create a new VM? (y/N): " confirm < /dev/tty
+  confirm=$(echo "$confirm" | tr '[:upper:]' '[:lower:]')
+  if [[ "$confirm" != "y" && "$confirm" != "yes" ]]; then
+    echo "Aborted. No new VM was created."
+    exit 0
+  fi
+fi
+
+# --- Choose VM name ---
+default_vm_name="gcns-vm"
```

Comment:
What does gcns stand for? is this google cloud night scout?
I think nightscout would be a better name